### ISSUE TITLE
Corruption when creating a pivot table where some cells are blank

### ISSUE THREAD

**Issue by ses4j** [EPPlusSoftware/EPPlus#410]
Using EPPlus 5.6.4, net5.0.

I'm trying to create a Pivot Table, but my source data has some blank cells, and that seems to corrupt the pivotCache somehow.  (If I have NO blank cells, it works fine.)  When I try to open the output file, it says:

> We found a problem with some content in 'test-out.xlsx'.  Do you want us to try to recover as much as we can?  If you trust the source of this workbook, click Yes.

After clicking Yes, it says:

> Repaired Records: PivotTable report from /xl/pivotCache/pivotCacheDefinition1.xml part (PivotTable cache)

And then it works fine.

Input file:
[test-in.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6634380/test-in.xlsx)
![image](https://user-images.githubusercontent.com/435529/121596985-24c25300-ca0e-11eb-9a8d-3552e7f2b07b.png)

Output (before repair):
[test-out.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6634387/test-out.xlsx)

What it looks like after I repair, which I think is correct:
![image](https://user-images.githubusercontent.com/435529/121596957-1c6a1800-ca0e-11eb-88f5-3bd0371a1236.png)

Test Program.cs:
```
using System;
using System.IO;
using System.CommandLine;
using OfficeOpenXml;
using OfficeOpenXml.Table;
using OfficeOpenXml.Table.PivotTable;

namespace addpivot
{
    public class Class1
    {
        static void Main()
        {
            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
            using (var package = new ExcelPackage(new FileInfo("test-in.xlsx")))
            {
                var wb = package.Workbook;
                var worksheet = wb.Worksheets.Add("Pivot Tables");
                var table = wb.Worksheets[0].Tables["Table1"];
                ExcelPivotTable pt = worksheet.PivotTables.Add(worksheet.Cells["A1"], table, "PT1");
                pt.RowFields.Add(pt.Fields["ColC"]);
                pt.DataFields.Add(pt.Fields["ColB"]);
                package.SaveAs(new FileInfo("test-out.xlsx"));
            }
        }
    }
}
```

**Comment by JanKallman** [comment]
Thanks, I'll have a look at it.

**Comment by JanKallman** [comment]
containsMixedTypes attribute is incorrectly set to true when a cache field contains null's and numbers -
(DataTypeFlags.Int | DataTypeFlags.Number | DataTypeFlags.Empty). Will supply a fix shortly

**Comment by ses4j** [comment]
@JanKallman Thanks!  Everything else works perfectly, and it's so much nicer than Aspose, which we had been trying.  As soon as this works we'll be Purchasing.  Much appreciate your work!

**Comment by JanKallman** [comment]
Thanks! You can try it with the latest Nuget package from our develop feed...https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed. It will go into version 5.7 released later this week.

**Comment by ses4j** [comment]
Works great, thank you so much!  Goodbye, Aspose!

**Comment by JanKallman** [comment]
Fixed in EPPlus 5.7.0
